CHANGELOG.md

```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Preview Release 5.2.0-preview1.23109.1] - 2023-04-19
+
+This update brings the below changes over the previous release:
+
+### Fixed
+
+- Fixed AE enclave retry logic for async queries. [#1988](https://github.com/dotnet/SqlClient/pull/1988)
+- Fixed activity correlator to continue use of same GUID for connection activity. [#1997](https://github.com/dotnet/SqlClient/pull/1997)
+- Fixed behavior when error class is greater than 20 on connection retry. [#1953](https://github.com/dotnet/SqlClient/pull/1953)
+- Fixed error message when symmetric key decryption failed using Always Encrypted. [#1948](https://github.com/dotnet/SqlClient/pull/1948)
+- Fixed TransactionScope connection issue when Enlist is enable, Pooling is disabled and network connection type is Redirect. [#1960](https://github.com/dotnet/SqlClient/pull/1960)
+- Fixed TDS RPC error on large queries in SqlCommand.ExecuteReaderAsync. [#1936](https://github.com/dotnet/SqlClient/pull/1936)
+- Fixed throttling of token requests by calling AcquireTokenSilent. [#1925] (https://github.com/dotnet/SqlClient/pull/1925)
+- Fixed Linux code coverage result in Build proj. [#1950] (https://github.com/dotnet/SqlClient/pull/1950)
+- Fixed NullReferenceException in GetBytesAsync. [#1906] (https://github.com/dotnet/SqlClient/pull/1906)
+- Fixed Transient fault handling issue with OpenAsync. [#1983] (https://github.com/dotnet/SqlClient/pull/1983)
+
+### Changed
+
+- Enabled SqlDiagnosticListener on .NET Standard. [#1931] (https://github.com/dotnet/SqlClient/pull/1931)
+- Replaced calls to Method.GetCurrentMethod().Name with nameof(). [#1943] (https://github.com/dotnet/SqlClient/pull/1943)
+- Upgraded Azure Identity from 1.7.0 to 1.8.0. [#1921] (https://github.com/dotnet/SqlClient/pull/1921)
+- Enhance invariant mode checks. [#1917] (https://github.com/dotnet/SqlClient/pull/1917)
+- Add and use TryReadChars method. [#1544] (https://github.com/dotnet/SqlClient/pull/1544)
+- Removed reference of Microsoft.Win32.Registry in netcore. [#1974](https://github.com/dotnet/SqlClient/pull/1974)
```

Comment:
```suggestion
- Removed reference on Microsoft.Win32.Registry since it's shipped starting with .NET 6.0. [#1974](https://github.com/dotnet/SqlClient/pull/1974)
```